core/templates/dev/head/pages/collection_editor/collection-editor-navbar-breadcrumb/collection-editor-navbar-breadcrumb.directive.ts

```diff
@@ -22,14 +22,15 @@
 // support both displaying the current title of the collection (or untitled if
 // it does not yet have one) or setting a new title in the case of an untitled
 // collection.
-oppia.directive('collectionEditorNavbarBreadcrumb', [
+
+angular.module('collectionEditorNavbarBreadcrumbModule').directive('collectionEditorNavbarBreadcrumb', [
```

Comment:
Break this too.

Reply:
Hi Yash, I will fix all the lint errors after the PR passes e2e tests as discussed in admin page PR.
Thanks!